sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/common/FileBasedIOTestPipelineOptions.java

```diff
@@ -48,4 +48,15 @@
   boolean getReportGcsPerformanceMetrics();
 
   void setReportGcsPerformanceMetrics(boolean performanceMetrics);
+
+  @Validation.Required
+  @Description("Expected hash for write correctness checking")
+  String getExpectedHash();
```

Comment:
This should probably eventually land in `IOTestPipelineOptions` interface but since you are working on file Based tests only in this PR I'm fine with the current state. This is just a note for the future. :)